sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -164,7 +167,7 @@ def __init__(  # pylint: disable=super-init-not-called
         self,
         feature_id: str,
         *,
-        enabled: Optional[bool] = None,
+        enabled: Optional[bool] = False,
```

Comment:
Is it still Optional?

Reply:
It is `Optional` in our API surface, though it's required in service call.

Reply:
> It is `Optional` in our API surface, though it's required in service call.

I don't quite get it. Can user set it to None?

Reply:
Users are not necessary to pass this parameter, it is optional for them. For type hint, I need to remove Optional to `bool = False` as None is not expected to set.

Reply:
In Python, optional does not mean "not require", it means "none-able" :)